config/environments/test.rb

```diff
@@ -22,8 +22,8 @@
 
   # Show full error reports and disable caching.
   config.consider_all_requests_local       = true
-  config.action_controller.perform_caching = false
-  config.cache_store = :null_store
+  config.action_controller.perform_caching = true
```

Comment:
@mark-cooper -- was not sure how to test a service class like `RecordCacheService` with the existing settings here.

Reply:
Figured this out...